src/main/java/io/vertx/core/net/TCPSSLOptions.java

```diff
@@ -405,6 +410,50 @@ public TCPSSLOptions addCrlPath(String crlPath) throws NullPointerException {
   }
 
   /**
+   * Add key options for SNI host.
+   *
+   * @param domain The domain name to match. May contain wildcards.
+   * @param keyCertOptions The key options to use when matching this host.
+   * @return a reference to this, so the API can be used fluently
+   */
+  public TCPSSLOptions addSNIKeyCertOptionsForDomain(String domain, KeyCertOptions keyCertOptions) throws NullPointerException {
```

Comment:
There are no getters for regular keyCertOptions other than getKeyCertOptions. I assume I can skip the getters for each type as well then, since it would have to be casted  back to the appropriate type.